src/mito2/src/manifest/storage.rs

```diff
@@ -264,12 +291,15 @@ impl ManifestObjectStore {
                     true
                 }
             })
-            .map(|e| e.0.path().to_string())
             .collect();
+        let paths = del_entries
+            .iter()
+            .map(|(e, _, _)| e.path().to_string())
+            .collect::<Vec<_>>();
         let ret = paths.len();
 
         debug!(
-            "Deleting {} logs from manifest storage path {} until {}, checkpoint: {:?}, paths: {:?}",
+            "Deleting {} logs from manifest storage path {} until {}, checkpoint_version: {:?}, paths: {:?}",
```

Comment:
```suggestion
            "Deleting {} logs from manifest storage path {} until {}, checkpoint_version: {}, paths: {:?}",
```

Reply:
`checkpoint_version` is `Option<&u64>` which doesn't impl `Display` trait.